storage-resize-images/extension.yaml

```diff
@@ -148,6 +148,24 @@ params:
     default: "200x200"
     required: true
 
+  - param: NAMING_CONVENTION
```

Comment:
If we make changes to the .yaml file then the extensions README.md also needs regenerating (see https://github.com/firebase/extensions/blob/next/storage-resize-images/functions/package.json#L13) and changes committed as part of this PR, thanks!